SRS.md

```diff
@@ -30,6 +30,21 @@ This project is an open-source project, and it is free for anybody who wishes to
 Crypto swap is a peer-to-peer cryptocurrency marketplace where buyers and sellers are connected for business, This platform provides a variety of rates for crypto sellers and well-secured payment methods, 
 Crypto-swae project we have chosen Vb.Net language for its more interactive support.
 
+### 1.5 REFERENCES
```

Comment:
References is not a place to mention names. 

Add an authors section at the top and add all contributors there. I would rather be listed as a contributor/supervisor.

Jam and the rest of the contributor should be mention and tagged with their respective roles.